app/models/pair_programming_exercise_feedback.rb

```diff
@@ -12,6 +12,32 @@ class PairProgrammingExerciseFeedback < ApplicationRecord
   scope :intermediate, -> { where.not(normalized_score: 1.00) }
   scope :final, -> { where(normalized_score: 1.00) }
 
+  enum comment: {
+    difficulty_easy: 0,
```

Comment:
Let's go with `too_easy`, since this would match the translation (and the `too_difficult`) better.